content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -0,0 +1,155 @@
+200
+200
+200
+{{< /text >}}
+
+## Configure clients to send mutual TLS traffic
+
+Configure Istio services to send mutual TLS traffic by setting `DestinationRule`.
+
+{{< text bash >}}
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "example-httpbin-istio-client-mtls"
+spec:
+  host: httpbin.foo.svc.cluster.local
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+{{< /text >}}
+
+`sleep.foo` and `sleep.bar` should start sending mutual TLS traffic to `httpbin.foo`. And `sleep.legacy` still sends plain text
+traffic to `httpbin.foo` since it does not have sidecar thus `DestinationRule` does not apply.
+
+Now we confirm all requests to `httpbin.foo` still succeed.
+
+{{< text bash >}}
+$ for from in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.foo: %{http_code}\n"; done
+200
+200
+200
+{{< /text >}}
+
+You can also specify a subset of the clients' request to use `ISTIO_MUTUAL` mutual TLS in
+[DestinationRule](https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule).
+After verifying it works by checking [Grafana to monitor](https://istio.io/docs/tasks/telemetry/using-istio-dashboard/),
+then increase the rollout scope and finally apply to all Istio client services.
+
+## Lock down to mutual TLS (optional)
+
+After migrating all clients to Istio services, injecting Envoy sidecar, we can lock down the `httpbin.foo` to only accept mutual TLS traffic.
+
+{{< text bash >}}
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "example-httpbin-permissive"
+  namespace: foo
+spec:
+  targets:
+  - name: httpbin
+  peers:
+  - mtls:
+      mode: STRICT
+EOF
+{{< /text >}}
+
+Now you should see the request from `sleep.legacy` fails.
+
+{{< text bash >}}
+$ for from in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.foo: %{http_code}\n"; done
+200
+200
+503
+{{< /text >}}
+
+If you can't migrate all your services to Istio (injecting Envoy sidecar), you have to stay at `PERMISSIVE` mode.
+However, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for the plain text traffic by default.
+We recommend to use [RBAC](https://istio.io/docs/tasks/security/role-based-access-control/) to configure different paths with different authorization policies.
```

Comment:
All links to content on the web site shojld just not use https://istio.io. Just start with /

Reply:
done.